plugins/modules/notification/discord.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Christian Wollinger <cwollinger@web.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: discord
+short_description: Send discord messages
```

Comment:
```suggestion
short_description: Send Discord messages
```
You sometimes write it with lower case D, and somtimes with upper case D. Since the official spelling is with upper case D, could you please change the documentation to always use upper case D?